packages/providers/src/mina-node/utils/custom-fetch.ts

```diff
@@ -0,0 +1,18 @@
+import JSONbig from 'json-bigint'
+
+export const customFetch = async (
```

Comment:
Is adding a custom fetcher for `graphql-request` actually needed? Sorry, I don't get the purpose.